src/main/java/sirius/biz/tenants/jdbc/SQLTenantAutoSetup.java

```diff
@@ -40,24 +47,35 @@ public void setup() throws Exception {
         }
 
         AutoSetup.LOG.INFO("Creating system tenant....");
-        SQLTenant tenant = new SQLTenant();
-        setupTenantData(tenant);
-        oma.update(tenant);
 
+        // The ID of the future system tenant will be set to "1" or another fixed value from the configuration files.
+        // This should normally be avoided at all cost, but in this case, it greatly simplifies the system config...
+        long systemTenantId = 1;
         if (tenants != null) {
-            try {
-                oma.updateStatement(SQLTenant.class)
-                   .set(SQLTenant.ID, Long.parseLong(tenants.getSystemTenantId()))
-                   .where(SQLTenant.ID, tenant.getId())
-                   .executeUpdate();
-            } catch (Exception e) {
-                AutoSetup.LOG.WARN("Failed to update ID of system tenant from %s to %s: %s",
-                                   tenant.getId(),
-                                   tenants.getSystemTenantId(),
-                                   e.getMessage());
-            }
+            systemTenantId = Long.parseLong(tenants.getSystemTenantId());
         }
```

Comment:
The new, mongo-like way to determine the ID is different from the old implementation (unless OMA assigns ID incrementally and starts at 1 anyway?) Is this what we want to have?